### DISCUSSION TITLE
Add password sign up/ sign in

### DISCUSSION THREAD

**Discussion by Pjaijai** [Pjaijai/Referalah#67] [投票 | Polls]
Using a magic link has been convenient for me to manage the project since it simplifies the process, and I've been avoiding the need to create a user interface for the pages. However, it appears that there's a growing interest in allowing people to use passwords for signing up and signing in. What should we do if the majority of you vote in favor of this change :

 - Create sign up page UI
 - Create sign in page UI
 - Create forget password page UI

The auth handing ca just pass it to supabase. Focusing on UI is fine (I guess)

Cut-off time: 20/10/2023 (Toronto time)

**Comment by yaucp** [comment]
I think we should just use apple/google for oauth login. It's more secure in this way and password management is quite annoying imo

**Reply by Pjaijai** [reply]
Sounds good ! Added option
